.ci/test.sh

```diff
@@ -2,4 +2,6 @@
 
 set -e
 
+python3 -c "from PIL import Image"
```

Comment:
Should we also do this in test-docker.yml and test-windows.yml?

Reply:
Fair enough. I've pushed a commit for test-windows.yml - but the change doesn't go in test-docker.yml, so I've created https://github.com/python-pillow/docker-images/pull/104